config/crds/migration_v1alpha1_migcluster.yaml

```diff
@@ -28,8 +28,8 @@ spec:
           type: object
         spec:
           properties:
-            clusterRef:
-              type: object
+            certificateAuthority:
```

Comment:
This may be better suited for a ConfigMap or Secret. I also wonder if `caBundle` is a more idiomatic var name for this type of thing inside of k8s, feel like I've seen that places before but I could be off base.

Reply:
I appreciate the simplicity of keeping this directly on the spec but don't know if it would make the MigCluster unwieldy or if there are size limitations. Simpler would be preferred if it won't be a burden (e.g. keeping it on the MigCluster).

Reply:
Yeah, looking through k8s code it seems `caBundle` is commonly used and is a `[]byte`.  Changing to match.